sdk/forms/src/main/java/com/klaviyo/forms/overlay/OverlayPresentationManager.kt

```diff
@@ -0,0 +1,6 @@
+package com.klaviyo.forms.overlay
+
+interface OverlayPresentationManager {
+    fun presentOverlay()
+    fun dismissOverlay()
+}
```

Comment:
Simple interface right now, this will get a little more responsibility when we convert to displaying the form in an activity.